wallet/src/wallet.rs

```diff
@@ -196,6 +201,11 @@ impl<'a, Backend: CapeWalletBackend<'a> + Sync + 'a> CapeWalletExt<'a, Backend>
         let mut state = self.lock().await;
 
         let erc20_code = state.backend().get_wrapped_erc20_code(&cap_asset).await?;
+        if erc20_code == default_erc20_code() {
```

Comment:
I think we should do this check in `CapeWalletBackend::get_wrapped_erc20_code`, and document in the trait definition that `get_wrapped_erc20_code` returns an error (or possibly `None`?) if the asset is not a wrapped asset.